animal.csl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>animal</title>
+    <id>http://www.zotero.org/styles/animal</id>
+    <link href="http://www.zotero.org/styles/animal" rel="self"/>
+    <link href="http://www.zotero.org/styles/south-african-journal-of-geology" rel="template"/>
+    <link href="http://www.animal-journal.eu/instructions_to_authors.htm" rel="documentation"/>
```

Comment:
@POBrien333, could you check whether these instructions are the same: https://www.cambridge.org/core/journals/animal/information/instructions-contributors

http://www.animal-journal.eu/Table%20of%20content.htm only goes up to 2012, whereas https://www.cambridge.org/core/journals/animal/all-issues goes up to 2017, so the Cambridge website seems more up to date.